test/integration/main_test.go

```diff
@@ -125,14 +125,38 @@ func TestMappingEventHandling(t *testing.T) {
 		wantProcessErrSubStr string
 	}{
 		{
-			name:              "mapping_success_event",
-			resourceName:      fmt.Sprintf("//storage.googleapis.com/%s", cfg.GCSStaticBucket),
+			// This test focus on testing resources that has projects/project_id in it's name.
+			name:              "mapping_success_event_ci",
```

Comment:
I think you'd better name the test case as `mapping_success_event` and `mapping_success_event_with_customized_scope`